### DISCUSSION TITLE
Unable to fork repositories

### DISCUSSION THREAD

**Discussion by Arzumify** [community/community#122125] [Repositories]
### Select Topic Area

Bug

### Body

I am not able to fork any repositories in GitHub, and it returns an error page stating the storage backend is offline. There is no maintenance happening on the status page linked.

**Comment by charlestanev** [comment]
Hey everyone,

I noticed Arzumify's post about the inability to fork repositories due to an error with the storage backend. That sounds like a frustrating issue! Has anyone encountered a similar problem recently? Let's see if we can collaborate to troubleshoot and resolve this issue together.

Thanks,
Charles
